packages/ui/src/utils/sign-in-experience.ts

```diff
@@ -27,21 +27,16 @@ const getSecondarySignInMethod = (signInMethods: SignInMethods) =>
     return methods;
   }, []);
 
-const getSignInExperienceSettings = async () => {
-  try {
-    const result = await getSignInExperience();
-    const settings = {
-      branding: result.branding,
-      languageInfo: result.languageInfo,
-      termsOfUse: result.termsOfUse,
-      primarySignInMethod: getPrimarySignInMethod(result.signInMethods),
-      secondarySignInMethods: getSecondarySignInMethod(result.signInMethods),
-    };
-
-    return { settings };
-  } catch (error: unknown) {
-    return { error };
-  }
+const getSignInExperienceSettings = async (): Promise<SignInExperienceSettings> => {
+  const { branding, languageInfo, termsOfUse, signInMethods } = await getSignInExperience();
+
+  return {
+    branding,
+    languageInfo,
+    termsOfUse,
+    primarySignInMethod: getPrimarySignInMethod(signInMethods),
+    secondarySignInMethods: getSecondarySignInMethod(signInMethods),
```

Comment:
nit: simplier
```ts
  const { signInMethods, ...rest } = await getSignInExperience();

  return {
    ...rest,
    primarySignInMethod: getPrimarySignInMethod(signInMethods),
    secondarySignInMethods: getSecondarySignInMethod(signInMethods),
  };
```

Reply:
As this function did tweak the return result, prefer to keep all fields visible.